Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM jupyter/datascience-notebook
 
-MAINTAINER Pierre Bellec <pierre.bellec@gmail.com>
+LABEL maintainer="Pierre Bellec <pierre.bellec@gmail.com>"
 
 USER jovyan
 
-RUN pip install nilearn
+# Copying the repository inside the container
+COPY . /home/jovyan/work
```

Comment:
So it looks like here you're assuming the build is on your local system. Can we have this as a relative path instead ? Maybe assuming that the Dockerfile is in the same directory as the code base ?

Reply:
I don`t get what you are asking for ?
With this command Docker will copy the current director `.` in `/home/jovyan/work`.
What do you mean by relative path ?

Reply:
Do you mean using 
```
RUN wget https://github.com/brainhack101/introML/archive/0.1.zip
RUN unzip 0.1.zip
RUN rm 0.1.zip`
```
like before ?

Reply:
And usually it is a best practice to build from the current directory. But I am not sure which solution is better with Binder

Reply:
Ok, sorry, I had forgotten that the `jovyan` user was Jupyter's convention !
Yes, I think we should build from the current directory, but I'm not sure why we're building in the `work` directory. Can we put the current working directory in the root folder ? This ties into [our conversation below](https://github.com/brainhack101/introML/pull/3/files#r251029589).

Reply:
I updated the file so now everything is inside `/home/jovyan` and no need to add the option `--notebook-dir` because jupyter create a server automatically inside the `/home`